### PR TITLE
health docs copy update

### PR DESCRIPTION
**Pull request by jasalisbury**
https://github.com/department-of-veterans-affairs/vets-contrib/issues/2815

Update health API copy per feedback from the health team. One thing I can't figure out is how to get the URLs on the health quickstart page to show up as links. I've tried using `[]()` syntax but the link target doesn't work properly when using the environment variable. Using `< link >` doesn't work either. If anyone has ideas let me know.

See [HealthApiDocumentationUpdated2.docx](https://github.com/department-of-veterans-affairs/developer-portal/files/3447535/HealthApiDocumentationUpdated2.docx) for the list of changes.

### CODE REVIEW

File: src/content/apiDocs/health/communityCareApiIntro.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-VA’s Community Care Eligibility API utilizes VA’s Facility API, VA's Enrollment & Eligibility system and others to satisfy requirements found in the VA's MISSION Act of 2018. Please visit VA's Official MISSION Act of 2018 page for more information about the specific requirements.
+VA’s Community Care Eligibility API utilizes VA’s Facility API, VA's Enrollment & Eligibility system and others to satisfy requirements found in the VA MISSION Act of 2018. Please visit the official [VA MISSION Act of 2018](https://missionact.va.gov/) page for more information about the specific requirements.
 
 The Community Care Eligibility API allows consumers to provide Veteran’s the ability to know if they meet the following MISSION Act requirements:
```
Comment (mjlumetta) [inline src/content/apiDocs/health/communityCareApiIntro.mdx]:
```suggestion
The Community Care Eligibility API allows consumers to provide Veterans the ability to know if they meet the following MISSION Act requirements:

```

**Comment by jasalisbury** [comment]
The links on the health quickstart page do not need to be clickable, so this PR is ready to be reviewed as is.

File: src/content/apiDocs/health/fhirR4ApiIntro.mdx [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ## R4
 
-FHIR is based on emerging industry approaches to better share information (i.e., interoperability). HL7's FHIR R4 includes the most recent updates and expansions to existing resources, including new implementations and workflows. The following describes the VA's implementation of HL7's FHIR R4 standard.
\ No newline at end of file
+This [CoverageEligibilityResponse](https://www.hl7.org/fhir/r4/coverageeligibilityresponse.html) resource is compliant with FHIR version R4 standards, more information regarding VA’s implementation of this resource can be found in the [Urgent Care API](/explore/health/docs/urgent_care).
```
Comment (mjlumetta) [inline src/content/apiDocs/health/fhirR4ApiIntro.mdx]:
```suggestion
This [CoverageEligibilityResponse](https://www.hl7.org/fhir/r4/coverageeligibilityresponse.html) resource is compliant with FHIR version R4 standards; more information regarding VA’s implementation of this resource can be found in the [Urgent Care API](/explore/health/docs/urgent_care).
```

or period and a new sentence

File: src/content/apiDocs/health/healthQuickstart.mdx [hunk 3]
```diff
@@ -17,7 +21,18 @@ The following basic information should be provided:
...
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/argonaut/data-query/`
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/dstu2/`
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/r4/`
+ * `{{ process.env.REACT_APP_API_URL }}/services/community-care/v0/eligibility`
+
+Accordingly, the FHIR conformance statements can be retrieved from:
+
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/argonaut/data-query/metadata`
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/dstu2/metadata`
+ * `{{ process.env.REACT_APP_API_URL }}/services/fhir/v0/r4/metadata`
```
Comment (mjlumetta) [inline src/content/apiDocs/health/healthQuickstart.mdx]: if these don't have to be links, then this is good from my point of view

**Pull request merged**